blog_system/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-75dk=yot+&l6u_lr+$*684z8o)0k6ahrq9g%h@%@r#^w$8d6nm'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Debug mode is currently turned on. This should never be the case in a production environment as it can expose sensitive information. Make sure to turn it off when deploying your app.